docs/developer/README.md

```diff
@@ -99,65 +99,72 @@ Integration tests use the `envtest` package from controller-runtime.  See this [
 **Note:** If you want to run integration tests from your IDE you need to set the `KUBEBUILDER_ASSETS` env var. It should point to `<project-root>/testbin/bin`.
 
 ## Running e2e tests
-End-to-end tests require kind clusters that are built with the `scripts/setup-kind-multicluster.sh` script. 
+End-to-end tests run against local kind clusters. There are two possible setups:
+* "multi-cluster" tests require two kind clusters;
+* "single-cluster" tests only need one kind cluster.
 
-**Note:** There are plans to add the ability to run the tests against other clusters. This is being tracked in [#112](https://github.com/k8ssandra/k8ssandra-operator/issues/112).
```

Comment:
That issue is closed, as well as a few follow-ups (#446, #463, #515?) but I haven't found a good summary on how to run against other clusters.

Reply:
@adutra, are there docs somewhere that @olim7t could refer to here?

Reply:
OK, this is ready to merge then.